lib/core/ast_traversal/transpile_frontend_action.cpp

```diff
@@ -29,6 +27,7 @@ std::unique_ptr<ASTConsumer> TranspileFrontendAction::CreateASTConsumer(Compiler
 
         compiler.getPreprocessor().addPPCallbacks(std::move(callback));
     }
+    compiler.getDiagnostics().setShowColors(true);
```

Comment:
We can't just force Color codes into output.

Reply:
Do you have any idea how to actually enable colors? This option didn't work :disappointed: